third_party/2and3/pynamodb/indexes.pyi

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Optional
+
+class IndexMeta(type):
+    def __init__(cls, name, bases, attrs) -> None: ...
+
+class Index:
```

Comment:
missing the metaclass. I think `metaclass=IndexMeta` should work in stubs in both 2 and 3, but haven't tried.